packages/devtools_app/test/shared/table_test.dart

```diff
@@ -1120,6 +1262,22 @@ class TestData extends TreeNode<TestData> {
   }
 }
 
+class PinnableTestData implements PinnableListEntry {
+  PinnableTestData({
+    required this.name,
+    required this.enabled,
+  });
+
+  final String name;
+  bool enabled;
```

Comment:
should this be final?

Reply:
Was planning on modifying it in place before but didn't end up doing that. Done.